projects/storefrontlib/src/cms-components/user/register/register.component.ts

```diff
@@ -43,9 +44,10 @@ export class RegisterComponent implements OnInit, OnDestroy {
   // TODO(issue:4237) Register flow
   isNewRegisterFlowEnabled: boolean =
     this.featureConfig && this.featureConfig.isLevel('1.1');
-  // TODO(issue:4989) Anonymous consents - remove
+
   isAnonymousConsentEnabled =
```

Comment:
type like above, (explicitly vs implicit) ;p

Reply:
As this is a named feature, it shouldn't be remove in the future, so no need to write a comment. That comment is only for the feature level toggles that need to be cleaned up in the future releases.